vizro-core/tests/unit/vizro/models/test_dashboard.py

```diff
@@ -195,6 +195,29 @@ def test_page_registry_with_images(self, page_1, mocker, tmp_path):
             layout=mocker.ANY,  # partial call is tricky to mock out so we ignore it.
         )
 
+    @pytest.mark.parametrize(
+        "logo_path, logo_dark_path, logo_light_path, raise_error",
+        [
+            ("logo.svg", "logo_dark.svg", "logo_light.svg", True),
+            ("logo.svg", None, "logo_light.svg", True),
+            ("logo.svg", "logo_dark.svg", None, True),
+            (None, None, "logo_light.svg", True),
+            (None, "logo_dark.svg", None, True),
+            (None, "logo_dark.svg", "logo_light.svg", False),
+            ("logo.svg", None, None, False),
+            (None, None, None, False),
+        ],
+    )
+    def test_validate_logos(self, page_1, tmp_path, logo_path, logo_dark_path, logo_light_path, raise_error):
+        # TODO: Add some logic that places the logos in the correct folder. How?
+        # I guess it needs to be similar to test_infer_image, but need further explanations.
+
```

Comment:
@antonymilne - there is some logic missing here to get this properly working, but I imagine that it will look similar to this draft? I took a look at the `test_infer_image` but I don't fully understand how you can mock the placement of these logos 🤔  So maybe we can discuss this one in our catch-up. That's the only piece missing for now 👍